### PR TITLE
fix(admin): contribution page

### LINKED ISSUE #2741: 🐛 [Bug] messagesCount always 0
## 🐛 Bugreport
in the admin area, there is no more visual representation when a moderator has written. it is because all messagesCount 0. if I write a message to a user then there is a number. but if I reload ... the value is again 0. 
but there are messages in it. 

https://user-images.githubusercontent.com/1324583/221191003-33c77568-2128-49bb-8b90-67a4eabd4047.mp4

### PR DESCRIPTION
**Pull request by Elweyn**
### Issues

- relates #2787

### Todo

- [X] None

### CODE REVIEW

File: admin/src/locales/de.json [hunk 1]
```diff
@@ -137,7 +135,9 @@
   },
   "redeemed": "eingelöst",
   "removeNotSelf": "Als Admin/Moderator kannst du dich nicht selber löschen.",
+  "reopen": "Reopen",
   "save": "Speichern",
+  "state": "Status",
```
Comment (Mogge) [inline admin/src/locales/de.json:140]:
```suggestion
  "status": "Status",
```

File: admin/src/locales/en.json [hunk 2]
```diff
@@ -137,7 +135,9 @@
   },
   "redeemed": "redeemed",
   "removeNotSelf": "As an admin/moderator, you cannot delete yourself.",
+  "reopen": "Reopen",
   "save": "Speichern",
+  "state": "State",
```
Comment (Mogge) [inline admin/src/locales/en.json:140]:
```suggestion
  "status": "State",
```

File: admin/src/pages/CreationConfirm.vue [hunk 3]
```diff
@@ -308,12 +319,12 @@ export default {
...
             },
           },
-          { key: 'deletedBy', label: this.$t('mod') },
+          { key: 'deletedBy', label: this.$t('moderator') },
           { key: 'chatCreation', label: this.$t('chat') },
         ],
         [
           // all contributions
-          { key: 'state', label: 'state' },
+          { key: 'state', label: this.$t('state') },
```
Comment (Mogge) [inline admin/src/pages/CreationConfirm.vue:327]:
```suggestion
          { key: 'state', label: this.$t('status') },
```

File: admin/src/locales/de.json [hunk 4]
```diff
@@ -137,7 +135,9 @@
   },
   "redeemed": "eingelöst",
   "removeNotSelf": "Als Admin/Moderator kannst du dich nicht selber löschen.",
+  "reopen": "Reopen",
```
Comment (Mogge) [inline admin/src/locales/de.json:138]: Can we find a German word here? May be just `Öffnen`

**Review by Mogge — CHANGES_REQUESTED** [review]

File: admin/src/locales/de.json (hunk 4 above)
Comment (Elweyn) [reply admin/src/locales/de.json:138]: ``neu eröffnen`` · ``wieder eröffnen`` · ``wiedereröffnen``

**Comment by Elweyn** [comment]
@Mogge I did change from ``state`` to ``status`` but I don't like the change cause for me it is a german description.

File: admin/src/locales/en.json [hunk 5]
```diff
@@ -149,7 +147,7 @@
     "totalGradidoUnbookedDecayed": "Unbooked GDD decay",
     "totalUsers": "Members"
   },
-  "status": "Status",
+  "status": "State",
```
Comment (Mogge) [inline admin/src/locales/en.json:150]:
```suggestion
  "status": "Status",
```

File: admin/src/pages/CreationConfirm.vue [hunk 6]
```diff
@@ -195,11 +206,11 @@ export default {
             key: 'contributionDate',
             label: this.$t('created'),
             formatter: (value) => {
-              return this.$d(new Date(value), 'short')
+              return value ? this.$d(new Date(value), 'short') : '-'
```
Comment (Mogge) [inline admin/src/pages/CreationConfirm.vue:209]: Can this be a method? Please have an em-dash as false: `'—'`

**Review by Mogge — CHANGES_REQUESTED** [review]

File: admin/src/pages/CreationConfirm.vue [hunk 7]
```diff
@@ -172,6 +184,9 @@ export default {
       this.items.find((obj) => obj.id === id).messagesCount++
       this.items.find((obj) => obj.id === id).state = 'IN_PROGRESS'
     },
+    formatDateOrDash(value) {
+      return value ? this.$d(new Date(value), 'short') : '-'
```
Comment (Mogge) [inline admin/src/pages/CreationConfirm.vue:188]:
```suggestion
      return value ? this.$d(new Date(value), 'short') : '—'
```

**Review by Mogge — CHANGES_REQUESTED** [review]

**Review by Mogge — DISMISSED** [review]

**Pull request merged**